app/networks/genesis.json

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
Why is there this genesis.json now?

Reply:
not entirely sure where that came from. I replaced my genesis.json with the one you sent me last week. Does this one not match that one?

Reply:
The location is odd, as genesis.json usually belongs in a folder for a network.

Reply:
aha, maybe i had some sloppy drag and drop happening! * removing *